glotaran/builtin/io/folder/folder_plugin.py

```diff
@@ -74,6 +137,15 @@ def save_result(
         ValueError
             If ``result_path`` is a file.
         """
+        if used_inside_of_plugin is not True:
+            warn(
+                UserWarning(
+                    "The folder plugin is only intended for internal use by other plugins "
+                    "as quick way to save most of the files. The saved result will be incomplete "
+                    ", thus it is not recommended to be used directly."
```

Comment:
In general, better move punctuation like `,` to end of a line, now it's hard to see that you have an extra space 
`UserWarning: The folder plugin is only intended for internal use by other plugins as quick way to save most of the files. The saved result will be incomplete , thus it is not recommended to be used directly.`

Reply:
Fixed in 72ca809